docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -230,6 +230,13 @@ All named curves are providers of :class:`EllipticCurve`.
     SECG curve ``sect283k1``. Also called NIST K-283.
 
 
+.. class:: SECT239K1
+
+    .. versionadded:: 1.2
+
+    SECG curve ``sect239k1``.
```

Comment:
I actually don't have a specific need for them and could remove them. However, they were listed in http://www.secg.org/sec2-v2.pdf along with all of the other curves already defined here and they were supported by OpenSSL, so I didn't really understand why they were excluded. I added them for completeness.

If you'd prefer to stick with the original list, I'm fine with dropping them.